package.json

```diff
@@ -48,8 +48,8 @@
     "pretest": "npm run clean",
     "test": "npm run gen:samples && npm run compile:samples && npm run lint:samples && npm run lint",
     "gen:samples": "node test/test.js --cache --trace",
-    "compile:samples": "tsc output/arithmetics.ts output/css.ts output/json.ts output/javascript.ts output/st.ts output/arithmetics-typed.ts",
-    "lint:samples": "tslint output/*.ts",
+    "compile:samples": "tsc --declaration output/arithmetics.ts output/css.ts output/json.ts output/javascript.ts output/st.ts output/arithmetics-typed.ts",
+    "lint:samples": "tslint `find output -not -iname *.d.ts -iname *.ts`",
```

Comment:
instead of this we could add the following to the tslint.json file, which is supported since v5.8

```json
"linterOptions": {
      "exclude": [
          "output/*d.ts"
      ]
  }
```